agent/lib/kontena/network_adapters/ipam_client.rb

```diff
@@ -73,7 +73,12 @@ def release_address(network, address)
       debug "response: #{response.status}/#{response.body}"
       JSON.parse(response.body)
     rescue Excon::Errors::HTTPStatusError => error
-      handle_error_response(error)
+      if error.response.status == 409
+        # IPAM return 409 in case the address still responds to ping, case zombies
+        warn error.response.body
```

Comment:
I was thinking that this is a bit of a special case and all the "real" errors would let the weave actor crash this way